wallet/reference/rpc-api.md

```diff
@@ -1,65 +1,28 @@
+toc_max_heading_level: 2
 ---
 
 # JSON-RPC API
 
 MetaMask uses the [`window.ethereum.request(args)`](provider-api.md#windowethereumrequestargs)
-provider method to wrap a JSON-RPC API.
+provider method to wrap a [JSON-RPC API](../concepts/rpc-api.md).
 The API contains standard Ethereum JSON-RPC API methods and MetaMask-specific methods.
 
 :::tip MetaMask API playground
 The RPC methods are documented in the interactive
-[MetaMask JSON-RPC API Playground](/wallet/reference/wallet_watchasset).
+[MetaMask JSON-RPC API Playground](/wallet/reference/eth_subscribe).
 :::
 
-Methods in the API playground may have the following tags:
-
-- **MetaMask** - These methods behave in ways specific to MetaMask, and may or may not be supported
-  by other wallets.
-  Some of these methods are documented in more detail on this page.
-- **Restricted** - These methods are [restricted](#restricted-methods), which require requesting
-  permission using [`wallet_requestPermissions`](#wallet_requestpermissions).
-- **Mobile** - These methods are only available on MetaMask Mobile.
-
 For more information on the standard Ethereum RPC methods, see the
 [Ethereum wiki](https://eth.wiki/json-rpc/API#json-rpc-methods).
 
-:::note
-All RPC method requests can return errors.
-Make sure to handle errors for every call to
-[`window.ethereum.request(args)`](provider-api.md#windowethereumrequestargs).
-:::
-
-## Restricted methods
-
-MetaMask introduced web3 wallet permissions in [EIP-2255](https://eips.ethereum.org/EIPS/eip-2255).
-In this permissions system, each RPC method is restricted or unrestricted.
-If a method is restricted, a dapp must request permission to call it using
-[`wallet_requestPermissions`](#wallet_requestpermissions).
-Under the hood, permissions are plain, JSON-compatible objects, with fields that are mostly used
-internally by MetaMask.
-
-Outside of [Snaps restricted methods](/snaps/reference/rpc-api/#restricted-methods), the only
-restricted method is
-[`eth_accounts`](/wallet/reference/eth_accounts), which
-allows you to access the user's Ethereum accounts.
-More restricted methods will be added in the future.
-
-## Unrestricted methods
-
-Unrestricted methods have no corresponding permission, but they might still rely on permissions to
-succeed (for example, the signing methods require calling the restricted
-[`eth_accounts`](/wallet/reference/eth_accounts) method),
-or they might require confirmation by the user (for example,
-[`wallet_addEthereumChain`](#wallet_addethereumchain)).
-
-The following are some MetaMask-specific unrestricted methods.
+The following are some MetaMask-specific [unrestricted methods](../concepts/rpc-api.md#unrestricted-methods).
 For the full list of MetaMask JSON-RPC API methods, see the
-[API playground](/wallet/reference/wallet_watchasset).
+[API playground](/wallet/reference/eth_subscribe).
 
-### eth_requestAccounts
+## eth_requestAccounts
 
 Requests that the user provide an Ethereum address to be identified by.
```

Comment:
This reads a bit ambiguous. I think the gist is: When a dapp calls this method, the dapp is asking the user to provide an Eth address that the dapp can access. So, a minor rewrite like this? "Requests the user to provide an Ethereum address." That should be enough. See [this](https://eips.ethereum.org/EIPS/eip-1102#concepts) for example.